### PR TITLE
#17835 full name for arrays

### LINKED ISSUE #17835: Wrong array creation DDL in export
### Description

When we export a table with array into the new table, the DDL isn't include info about array type.
e.x. export default.AUTO_TEST_DO_NOT_EDIT_EXPORT to a new table '**TEST_TABLE**':
```
CREATE TABLE `default`.TEST_TABLE (

	Column1 String,
	Column2 Array,  //should be Array(Int64)
	Column3 DateTime64,
	Column4 Date,
	Column5 Float64

) ENGINE = Log;
```

image

### DBeaver Version

22.2.1

### Operating System

win10

### Database and driver

Clickhouse 0.2.6

### Steps to reproduce

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Destrolaric**
Uses full name for the array datatype
Test for array datatype
Closes #17835 
image

### CODE REVIEW

File: plugins/org.jkiss.dbeaver.ext.clickhouse/src/org/jkiss/dbeaver/ext/clickhouse/model/ClickhouseTableColumn.java [hunk 1]
```diff
@@ -43,6 +43,10 @@ public String getFullTypeName() {
     @Override
     public void setFullTypeName(String fullTypeName) throws DBException {
         this.fullTypeName = fullTypeName;
+        if (fullTypeName.toUpperCase().contains("ARRAY")) {
+            setTypeName(fullTypeName);
+            return;
+        }
         super.setFullTypeName(fullTypeName);
```
Comment (E1izabeth) [inline plugins/org.jkiss.dbeaver.ext.clickhouse/src/org/jkiss/dbeaver/ext/clickhouse/model/ClickhouseTableColumn.java]: I don't see the reason to use `return` here when you have `else`.
```suggestion
        if (fullTypeName.toUpperCase().contains("ARRAY")) {
            setTypeName(fullTypeName);
        } else {
            super.setFullTypeName(fullTypeName);
        }
```

Comment (Destrolaric) [reply plugins/org.jkiss.dbeaver.ext.clickhouse/src/org/jkiss/dbeaver/ext/clickhouse/model/ClickhouseTableColumn.java]: Fixed

**Review by E1izabeth — APPROVED** [review]
Not sure whether we should do the same for other data types

**Comment by serge-rider** [comment]
Please attach screenshot with new DDL info

**Comment by E1izabeth** [comment]
Still no screenshot here.

**Comment by Destrolaric** [comment]
Attached the screenshot

**Review by LonwoLonwo — APPROVED** [review]

**Comment by Matvey16** [comment]
Verified

**Pull request merged**